scripts/encrypt_ami.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python3
+
+import boto3
+import sys
+
+
+def create_encrypted_ami_copy(ami_id, region='us-east-1', description='Encrypted copy of AMI'):
+    """
+    Create an encrypted copy of an AMI, copy its tags, add a new tag, and print the new AMI ID.
+
+    :param ami_id: The ID of the source AMI.
+    :param optional region: The region where the AMI is located. Default is 'us-east-1'.
+    :param optional description: A description for the new AMI.
+    """
+    # Create an EC2 resource session
+    ec2 = boto3.client('ec2', region_name=region)
+
+    try:
```

Comment:
The `try:` and the extra level of indentation through the rest of this function also need to be removed.